src/cascadia/TerminalSettingsEditor/Launch.xaml

```diff
@@ -114,15 +114,17 @@ the MIT License. See LICENSE in the project root for license information. -->
                                 Margin="0,0,10,0">
                         <TextBlock x:Uid="Globals_InitialCols" Style="{StaticResource LaunchSizeSubHeaderStyle}"/>
                         <muxc:NumberBox Value="{x:Bind State.Settings.GlobalSettings.InitialCols, Mode=TwoWay}"
-                                        Style="{StaticResource LaunchSizeNumberBoxStyle}"/>
+                                        Style="{StaticResource LaunchSizeNumberBoxStyle}"
+                                        Minimum="1"/>

```

Comment:
Please move setting the `minimum` value to the `LaunchSizeNumberBoxStyle` declaration on line 91.